Tests/TuistLintCodeIntegrationTests/EmptyTests.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import XCTest
+
+@testable import TuistDependencies
+@testable import TuistSupportTesting
+
+final class EmptyTests: TuistUnitTestCase {
```

Comment:
I'd probably for not including this file - we can add the integration tests target when necessary

Reply:
I wasn't sure how to handle this situation. 

I think it makes sense what you have proposed here. I will remove `TuistLintCodeIntegrationTests` target. 

I will also remove `TuistDependenciesIntegrationTests` target which is empty too.

Reply:
done